From 5b6a91770e356045b3ef755debbfb1588e0523bc Thu, 15 Dec 2016 17:28:54 +0100
From: Reto Zingg <reto.zingg@jollamobile.com>
Date: Thu, 30 May 2013 13:21:39 +0300
Subject: [PATCH] install data generation scripts and disable packaging of generated data


this allows to generate the data for tests to be build during rpm
installation instead of packaging in (~160MB)

[tracker] install the data generation scripts

diff --git a/utils/data-generators/cc/Makefile.am b/utils/data-generators/cc/Makefile.am
index c41494c..42b23ed 100644
--- a/utils/data-generators/cc/Makefile.am
+++ b/utils/data-generators/cc/Makefile.am
@@ -18,3 +18,15 @@
 	tracker.py                                     \
 	convert_data.py                                \
 	source-data
+
+nemotestsdir = /opt/tests/tracker/bin
+nemotests_DATA = default.cfg max.cfg
+nemotests_SCRIPTS = generate
+nemotests_PYTHON = gen_data.py __init__.py mfo.py mlo.py mto.py ncal.py nco.py nfo.py nmm.py \
+                   nmo.py slo.py ontology_prefixes.py tools.py tracker.py convert_data.py
+
+nemotestdatadir = /opt/tests/tracker/bin/source-data
+nemotestdata_DATA = source-data/area-codes.txt source-data/equipment-manufacturers.txt source-data/job-titles.txt source-data/male-prefix.txt \
+                    source-data/company-names.txt source-data/equipment-models.txt source-data/last-name.txt source-data/street-names.txt \
+                    source-data/company-types.txt source-data/female-first-name.txt source-data/latin-words.txt source-data/street-types.txt \
+                    source-data/email-domains.txt source-data/female-prefix.txt source-data/male-first-name.txt source-data/zip-codes.txt
diff --git a/tests/functional-tests/ttl/Makefile.am b/tests/functional-tests/ttl/Makefile.am
index 0c4d60e..2a9aac0 100644
--- a/tests/functional-tests/ttl/Makefile.am
+++ b/tests/functional-tests/ttl/Makefile.am
@@ -9,51 +9,3 @@
 
 ttldir = $(datadir)/tracker-tests/ttl
 
-ttl_DATA =                                             \
-	010-nco_EmailAddress.ttl                       \
-	011-nco_PostalAddress.ttl                      \
-	012-nco_PhoneNumber.ttl                        \
-	013-nco_IMAddress.ttl                          \
-	014-nco_ContactEmail.ttl                       \
-	015-nco_ContactCall.ttl                        \
-	016-nco_ContactIM.ttl                          \
-	018-nco_PersonContact.ttl                      \
-	020-slo_GeoLocation.ttl                        \
-	021-slo_Landmark.ttl                           \
-	026-mlo_GeoPoint.ttl                           \
-	027-mlo_LocationBoundingBox.ttl                \
-	028-mlo_GeoLocation.ttl                        \
-	029-mlo_Landmark.ttl                           \
-	030-nmo_MailAccount.ttl                        \
-	031-nmo_MailFolder.ttl                         \
-	032-nmo_Email.ttl                              \
-	035-nmo_CommunicationChannel.ttl               \
-	036-nmo_IMMessage.ttl                          \
-	037-nmo_SMSMessage.ttl                         \
-	038-nmo_Call.ttl                               \
-	040-nmm_Artist.ttl                             \
-	040-nmm_MusicAlbumDisc.ttl                     \
-	041-nmm_MusicAlbum.ttl                         \
-	042-nmm_MusicPiece.ttl                         \
-	045-nmm_Photo.ttl                              \
-	044-nfo_Equipment.ttl                          \
-	046-nmm_Video.ttl                              \
-	050-tracker_Volume.ttl                         \
-	051-nfo_PlainTextDocument.ttl                  \
-	060-nfo_SoftwareCategory.ttl                   \
-	061-nfo_SoftwareApplication.ttl                \
-	065-nfo_WebHistory.ttl                         \
-	070-ncal_Alarm.ttl                             \
-	071-ncal_Calendar.ttl                          \
-	072-ncal_Event.ttl                             \
-	073-ncal_Todo.ttl                              \
-	080-mfo_FeedChannel.ttl                        \
-	081-mfo_FeedMessage.ttl                        \
-	090-mto_TransferElement.ttl                    \
-	091-mto_UploadTransfer.ttl
-
-DISTCLEANFILES =                                       \
-	$(BUILT_SOURCES)                               \
-	$(ttl_DATA)                                    \
-	source-data.pkl
-
